packages/office-ui-fabric-react/src/components/Check/Check.styles.ts

```diff
@@ -0,0 +1,136 @@
+import { ICheckStyleProps, ICheckStyles } from './Check.types';
+import {
+  concatStyleSets,
+  getTheme,
+  HighContrastSelector,
+  IStyle,
+  ITheme,
+} from '@uifabric/styling';
+import { memoizeFunction } from '@uifabric/utilities';
+
+const DEFAULT_CHECKBOX_HEIGHT: string = '18px';
+
+export const getStyles = (
+  props: ICheckStyleProps
+): ICheckStyles => {
+  const {
+    checkBoxHeight = DEFAULT_CHECKBOX_HEIGHT,
+    theme,
+    checked
+  } = props;
+
+  const { palette, semanticColors } = theme;
+
+  const _sharedCircleCheck: IStyle = {
+    fontSize: checkBoxHeight,
+    position: 'absolute',
+    left: 0,
+    top: 0,
+    width: checkBoxHeight,
+    height: checkBoxHeight,
+    textAlign: 'center',
+    verticalAlign: 'middle'
+  };
+
+  return ({
+    root: [
+      {
+        // lineHeight currently needs to be a string to output without 'px'
+        lineHeight: '1',
```

Comment:
@dzearing apparently this is a bug where providing the type number automatically converts the value into a string with 'px' appended.

Reply:
Yup, there was a PR to fix it by Micheal but had to be reverted as some people are using lineHeight: 20 intending for it to output line-height: 20px.